Paralayout/GeometryAdditions.swift

```diff
@@ -255,3 +265,20 @@ public extension CGRect {
     }
 
 }
+
+
+// MARK: - UIEdgeInsets Convenience Methods
+
+
+public extension UIEdgeInsets {
+    
+    /// The combined top and bottom insets.
+    var verticalAmount: CGFloat {
```

Comment:
`Amount` suffix feels off to me. I might call this `height`, or use a suffix `Padding`?

Reply:
The top and bottom insets face opposite directions so "height" doesn't feel right (it's sort of the opposite of height--it's a height reduction). Repeating the word "inset" would be redundant, and just "vertical" seemed confusing.

Reply:
Fair. Naming is hard 😅.

Reply:
You should have heard the discussion that lead to "ScaleFactorProviding." 😝

Reply:
If we want to be more specific than just "amount", we could go with `verticalTotal` or `verticalSum` to be more specific.  Fine with it as is though.